src/main/java/core/basesyntax/service/impl/BalanceHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+
+public class BalanceHandler implements OperationHandler {
+    @Override
+    public void toProcess(FruitTransaction fruitTransaction) {
+        new StorageDaoImpl().create(
```

Comment:
you should create a global variable from StorageDao and initialize it using constructor, the same for other handlers